cmd/plotFunctions/testdata/expected.beta.txt

```diff
@@ -0,0 +1,101 @@
+X	f(X)
+0.001000	10.070879
```

Comment:
Are these numbers from a trusted source, such as R?  I am wondering what we will do if we miss a test by a little bit.  How do we know which (newer/older) is more accurate.